core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpSamDAO.scala

```diff
@@ -73,6 +69,40 @@ class HttpSamDAO(baseSamServiceURL: String, serviceAccountCreds: Credential)(imp
     }
   }
 
+  def listUserPoliciesForResource(resourceTypeName: SamResourceTypeName, resourceId: String, userInfo: UserInfo): scala.concurrent.Future[Set[SamPolicyWithName]] = ???
```

Comment:
there are a few unimplemented calls to sam as yet... will get back to them once implemented in sam